### DISCUSSION TITLE
Implementation of a Code of Conduct?

### DISCUSSION THREAD

**Discussion by FahimFBA** [ThioJoe/YT-Spammer-Purge#475] [Ideas]
Hello @ThioJoe, this project is currently missing the **Code of Conduct**. Shouldn't we add them as this project is getting noticed a lot and a lot of the users might want to get involved in it? 

For those who are not sure about the Code of Conduct, you can check the docs. Like the GitHub docs say,
> A code of conduct defines standards for how to engage in a community. It signals an inclusive environment that respects all contributions. It also outlines procedures for addressing problems between members of your project's community.

Let me know what do you think about that. If you want to add the Code of Conduct as well, then I want to add a PR.

**Comment by KendallDoesCoding** [comment]
I made a PR a while ago to add a COC (Code of Conduct), #103 , but as @ThioJoe stated:

Eh this isn't really necessary and the default template covers way too much stuff that doesn't apply. **This is a small project so I'll just be removing anything off topic anyway.** 

But since Linus's video, a lot of contributors have come in, to help as expected so yeah, but if its not too much and if you can remove stuff off-topic still, then yeah there's no need of this, but if you want one, feel free to re-open my PR (#103)
